src/types/conversation-update.ts

```diff
@@ -65,3 +37,30 @@ export class ConversationUpdate {
     });
   }
 }
+export interface IConversationParticipantFromEvent {
+  id: string;
+  purpose: string;
+  userId: string;
+  videos: Array<IParticipantVideo>;
+  calls: Array<IParticipantCall>;
+}
+
+export interface IParticipantVideo {
+  context: string;
+  audioMuted: boolean;
+  videoMuted: boolean;
+  id: string;
+  state: CommunicationStates;
+  peerCount: number;
+  sharingScreen: boolean;
+}
+
+export interface IParticipantCall {
+  id: string;
+  state: CommunicationStates;
+  muted: boolean;
+  confined: boolean;
+  held: boolean;
+  direction: 'inbound' | 'outbound';
+  provider: string;
+}
```

Comment:
These came over from my persistent connection work. Didn't mean to include them in this PR, but they aren't hurting anything.